### ISSUE TITLE
Error: Uncaught Error: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by Spidyisback** [iotaledger/firefly#2020]
Upgrade to 1.3.2 and staking doesn't work. After first time use the staking option is shown but doesn't work (error message: "Something went wrong"). When I restart Firefly I get the following message: 

App Version: 1.2.0

platform: win32
platformVersion: 10.0.19042
platformArchitecture: x64
cpuCount: 8
totalMem: 16258.7 MB
freeMem: 8974.9 MB
userPath: C:\Users\spidy\AppData\Roaming\Firefly

Error Type: Main Unhandled Promise Rejection

Error: Uncaught Error: ENOENT: no such file or directory, open 'C:\Users\spidy\AppData\Local\Programs\desktop\resources\app-update.yml'

![2022 01 05--Something-went-wrong-Github](https://user-images.githubusercontent.com/93836020/148195907-58fff689-baa2-4c1c-94b8-9e7cfdbfeaf8.jpg)

**Comment by MiKeF05** [comment]
Have you already updated to the latest version? Can you also share the error log of Firefly?

**Comment by rajivshah3** [comment]
In 1.3.1 we introduced the option to choose the installation path. When you installed the update, which path did you choose? Also, are you launching Firefly from a shortcut?

**Comment by Spidyisback** [comment]
Hi,
Thanks for your responses. Indeed the issue has been from the version(s). The installer run through, but later at second start of the app, it always displayed v1.2 instead of the newer 1.3.1 or 1.3.2. What I did now is delete the files in C:\Users\spidy\AppData\Roaming\Firefly\ manually, de-installed the app via Windows and then started all over again. With this (and setting up a new profile), I came to 1.3.2 and the funds can be staked now as well. It has been a bit weird, but essentially an update issue.
Will close the matter, thanks for your thoughts.
